com.unity.formats.alembic/Editor/Importer/AlembicImporter.cs

```diff
@@ -138,13 +141,17 @@ void OnValidate()
             }
         }
 
+        internal const string renderPipepineDependency = "AlembicRenderPipelineDependency";
+
         public override void OnImportAsset(AssetImportContext ctx)
         {
             if (ctx == null)
             {
                 return;
             }
 
+            ctx.DependsOnCustomDependency(renderPipepineDependency);
```

Comment:
Do you want to always depend on the custom render pipeline?
Is it possible a model doesn't have any material created from the default material and thus doesn't have to re-import on pipeline change?